assemblies/dynamic-plugins/assembly-installing-rhdh-plugins.adoc

```diff
@@ -6,6 +6,13 @@ The dynamic plugin support is based on the backend plugin manager package, which
 
 You can use the dynamic plugins that come preinstalled with {product} or install external dynamic plugins from a public NPM registry.
 
+[TIP]
+====
+In your plugins configuration file, for preinstalled plugins, use the path specified in the preinstalled plugins tables in the link:{plugins-ref-book-url}#rhdh-supported-plugins[{plugins-ref-book-title}]. 
+
+Use the plugin ID in your plugins' configuration file for external dynamic plugins that you download from an NPM registry.
```

Comment:
I'm not sure it's that simple. How does RHDH find your downloaded plugin? Doesn't it need to be in the cluster somehow for this to work? @batzionb what advice do you give to Orchestrator plugin users?

Reply:
I would consider providing an example for each scenario to make it crystal clear what changes have to be made.